util/util_test.go

```diff
@@ -905,3 +907,66 @@ func TestPathSliceSort(t *testing.T) {
 		}
 	}
 }
+
+func TestCopyFs1(t *testing.T) {
```

Comment:
Since we're testing a function in afero.go, please add the tests in afero_test.go